dts/bindings/mtd/partition.yaml

```diff
@@ -16,6 +16,17 @@ properties:
 child-binding:
     description: Flash partition child node
     properties:
+       mntauto:
+          type: boolean
+          required: false
+          description: if the partition should be mounted or not
+       mnttype:
+          type: int
+          required: false
+          description: file system identifier, as defined in fs/fs_type.h
+       mntpoint:
```

Comment:
I'd prefer `mount-point`, but then I'm not so supportive of rmvng vwls 2 rdc idntfr lngth.

Reply:
OK, will replace the name.

Reply:
> 
>  rmvng vwls 2 rdc idntfr lngth.

Eh I think I am removing vowels to much, I have failed to notice the sentence lacks many of them